.circleci/config.yml

```diff
@@ -166,12 +166,17 @@ commands:
     fetch_dependecies_version:
         description: "Fetch the dependencies version"
         steps:
+            - run:
+                  name: "Loads config enviroment variables at runtime"
+                  command: |
+                      tail -n +2 ./scripts/load-config.sh >> $BASH_ENV
```

Comment:
I assume load-config.sh couldn't just be `source`d here? I mean instead of appending it to `$BASH_ENV`

Reply:
That didn't work, no